Tiltfile

```diff
@@ -72,28 +91,15 @@ docker_build('words', 'words')
 docker_build('secrets', 'secrets')
 docker_build('sleep', 'sleeper')
 docker_build('sidecar', 'sidecar')
-
-# fast builds show how we can handle complex cases quickly
-(fast_build('fe',
-            'Dockerfile.go.base', '/go/bin/fe --owner ' + username)
-  .add(repo.path('fe'), '/go/src/github.com/windmilleng/servantes/fe')
-  .run('go install github.com/windmilleng/servantes/fe'))
-(fast_build('hypothesizer', 'Dockerfile.py.base')
-  .add(repo.path('hypothesizer'), '/app')
-  .run('cd /app && pip install -r requirements.txt', trigger='hypothesizer/requirements.txt'))
-(fast_build('fortune', 'Dockerfile.go.base')
-  .add(repo.path('fortune'), '/go/src/github.com/windmilleng/servantes/fortune')
-  .run('cd src/github.com/windmilleng/servantes/fortune && make proto')
-  .run('go install github.com/windmilleng/servantes/fortune'))
-
-# You can also ADD fast build instructions to a normal docker build; we use the fast build
-# instructions to update the live container when possible, otherwise use the docker build
-# instructions for a fresh build + deploy
-spoonerisms = docker_build('spoonerisms', 'spoonerisms')
-spoonerisms.add(repo.path('spoonerisms/src'), '/app')
-spoonerisms.add(repo.path('spoonerisms/package.json'), '/app/package.json')
-spoonerisms.add(repo.path('spoonerisms/yarn.lock'), '/app/yarn.lock')
-spoonerisms.run('cd /app && yarn install', trigger=['spoonerisms/package.json', 'spoonerisms/yarn.lock'])
+docker_build('spoonerisms', 'spoonerisms',
```

Comment:
Why is this live_update image separate from all of the others?

Reply:
lol no reason, i just followed the order of the original docker build declarations 😅 let me re-order